test/analyzers/lint_analyzer/rules/rules_list/prefer_const_border_radius/prefer_const_border_radius_test.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:dart_code_metrics/src/analyzers/lint_analyzer/rules/rules_list/prefer_const_border_radius/prefer_const_border_radius.dart';
+import 'package:test/test.dart';
+
+import '../../../../../helpers/rule_test_helper.dart';
+
+const _examplePath = 'prefer_const_border_radius/example';
+const _exampleClass = '$_examplePath/example.dart';
+
+void main() {
+  test('reports border const', () async {
```

Comment:
We prefer wrapping the test cases in a `describe` section, and name them `reports about found issues` if the test should report issues and reports no found issues` if it should not report issues. Please, fix it. Also, `initialization` test case is required to check the rule severity and name.